docs/source/constants.rst

```diff
@@ -11,5 +11,5 @@ To test Google's compliance with the GA4GH API, you can use the compliance tests
 http://ga4gh.org/#/compliance
```

Comment:
I don't think this link points to the compliance suite.  I believe this is the current Github one:

https://github.com/ga4gh/compliance

Probably Cassie's old test suite would probably need to be updated too:

http://cassiedoll.github.io/compliance-staging.html